h/routes.py

```diff
@@ -233,6 +233,12 @@ def includeme(config):  # noqa: PLR0915
         factory="h.traversal.GroupRequiredRoot",
         traverse="/{pubid}",
     )
+    config.add_route(
+        "group_moderation",
+        "/groups/{pubid}/moderation",
```

Comment:
I wonder if this should be `moderate` as a verb to align with `create`/`edit` etc? If the URL hasn't already been discussed with the backend folks, maybe start a thread in Slack. I don't think this needs to block merging this PR.